src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleViewModel.java

```diff
@@ -0,0 +1,89 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.errorconsole;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefGUI;
+import net.sf.jabref.gui.ClipBoardManager;
+import net.sf.jabref.gui.FXDialogs;
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+import net.sf.jabref.logic.error.MessagePriority;
+import net.sf.jabref.logic.error.ObservableMessageWithPriority;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.logging.ObservableMessages;
+import net.sf.jabref.logic.util.BuildInfo;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class ErrorConsoleViewModel {
+
+    private static final Log LOGGER = LogFactory.getLog(ErrorConsoleViewModel.class);
+    private final String REPORTISSUEURL = "https://github.com/JabRef/jabref/issues/new";
+    private final DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
+    private final Date date = new Date();
+
+    // handler for copy of Log Entry in the List by click of Copy Log Button
+    public void copyLog() {
+        ObservableList<ObservableMessageWithPriority> masterData = ObservableMessages.INSTANCE.messagesPropety();
+        masterData.forEach(message -> message.setIsFiltered(message.getPriority() != MessagePriority.LOW));
```

Comment:
i should get only the log entry or ? i must get this about a filter. Or how i should implement it ? 😄